dbt_subprojects/dex/seeds/trades/otsea_base_base_trades_seed.csv

```diff
@@ -0,0 +1,3 @@
+blockchain,project,version,block_month,block_date,block_time,block_number,token_sold_amount_raw,token_bought_amount_raw,token_sold_address,token_bought_address,maker,taker,project_contract_address,tx_hash,evt_index
+base,otsea,1,2024-05-01 00:00,2024-05-02 00:00,2024-05-02 07:41,13923182,88652482269503546099290,4000000000000000,0x9c1a0802fbe08bf7c68bb336820b7b2a7e4eb697,0x0000000000000000000000000000000000000000,0x9c1a0802fbe08bf7c68bb336820b7b2a7e4eb697,0xa836af59d35b3da3c9fb5cfd6f84d9e3bd8c5fd5,0xa836af59d35b3da3c9fb5cfd6f84d9e3bd8c5fd5,0xb6038c12bff786acb485fef9402d4215c97242a757c3f21ffebd0fe1aee7e92d,161
```

Comment:
Please verify the trade:
https://basescan.org/tx/0xb6038c12bff786acb485fef9402d4215c97242a757c3f21ffebd0fe1aee7e92d